develop/tutorials/articles/110-portlets/15-javascript/04-npm/08-npm-sdk.markdown

```diff
@@ -134,9 +136,11 @@ increased from version 1.1.0 to 1.5.0 for example):
 - Loader modules extender: 1.5.0
 - AMD loader: 2.1.0
 
-You are still in feature level 2, as all version changes have been semantical 
+You are still in feature level 2, as all version changes have been semantic 
 versioning compatible. If, however, you update to these version numbers:
-
+<!--How is a user supposed to know if his version changes are compatible? As
+far as I can tell, the npm SDK Assistant only tells the user the minimum
+version numbers.-->
```

Comment:
As long as you have the minimum version numbers, it is compatible with that feature set, meaning that you can use the features that are listed for that feature level. So, if your components meet the minimum requirements for a feature level, you can use those features and any features listed in the feature levels below it.